releasenotes/notes/queue-merge-started-at-b8efc3b8ecf348f5.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add ``queue-merge-started-at`` condition.
```

Comment:
Need another entry for `checks_timeout`.

Reply:
It already exists, in the other file.

Reply:
arf why you do 2 files omg 😁